pkg/runtime/runtime.go

```diff
@@ -685,9 +685,9 @@ func (a *DaprRuntime) subscribeTopic(parentCtx context.Context, name string, top
 
 			msg.Metadata[pubsubName] = name
 
-			rawPayload, serdeErr := contribMetadata.IsRawPayload(route.metadata)
-			if serdeErr != nil {
-				log.Errorf("error deserializing pubsub metadata: %s", err)
+			rawPayload, innerErr := contribMetadata.IsRawPayload(route.metadata)
```

Comment:
The linter caused these errors :) There were a mix of inner errors, but in lines below was still using `err`.

Reply:
ohh, my bad - still `err` was getting referred in log line.